pagerduty/resource_pagerduty_event_orchestration_path_global_test.go

```diff
@@ -291,6 +296,27 @@ func testAccCheckPagerDutyEventOrchestrationGlobalDefaultConfig(t, ep, s, o stri
 	`)
 }
 
+func testAccCheckPagerDutyEventOrchestrationGlobalDefaultWithEscalationPolicy(t, ep, s, o string) string {
+	return fmt.Sprintf("%s%s", createBaseGlobalOrchConfig(t, ep, s, o),
+		`resource "pagerduty_event_orchestration_global" "my_global_orch" {
+			event_orchestration = pagerduty_event_orchestration.orch.id
+
+			catch_all {
+				actions {}
+			}
+			set {
+				id = "start"
+				rule {
+						label = "rule 1"
+						actions {
+							 "escalation_policy": "POLICY"
```

Comment:
We need to get rid if the quotes around `escalation_policy` and use `=` instead of `:` because it's invalid syntax that causes the build to fail.